sigstore-java/src/main/resources/dev/sigstore/tuf/README.md

```diff
@@ -13,9 +13,9 @@ We keep copies of the remote tuf repositories locally in
    for interfacing with *.sigstage.dev
 
 For this client to function we need the following keys
-1. CTFE public key (`ctfe.pub` \
+1. CTFE public keys (`ctfe.pub`, `ctfe_2022.pub`) \
```

Comment:
why do we need the old key?

Reply:
you need the old key to verify that older SCTs embedded into older certs can still be verified (since they are not re-issued).

Reply:
also I need to update htis readme

Reply:
hmm, I guess we need the full history to verify old rekor entries with SCTs that were created with older keys.. I need to talk to Asra I think because I thought she said it was fine to just bundle the latest in the client.

Reply:
we need a full key history for all of sigstore (unless there is an expiry date for software?)

Reply:
Yeaah there's still a bit more to do here. We need to allow the client to accept multiple fulcioRoots and rekorPubKeys.